### ISSUE TITLE
Spatial position is floating not stable

### ISSUE THREAD

**Issue by chiangandy** [google-ar/arcore-unity-extensions#156]
Hello all,

   I am using the ARcore extension geospatial framework to build an app. briefly is working but sometime I fill spatial position is floating and not stable. situation like below:

1. I stand a place and use app to add anchor, anchor is stay position fine in space.
2. when I minimize app and re-active app, and do the resolve anchor,  sometime the re-draw anchor not stay in same position and move to others.
3. when I close app and lunch app again, anchor is stay in same position. 

Does everyone has this situation? Any information will be appreciated. :)

Andy Chiang

![Screenshot_20231124_172859_com womo womogo](https://github.com/google-ar/arcore-unity-extensions/assets/1468187/6df9319c-c046-4ee5-aa3f-d2cef972c358)
![Screenshot_20231124_172723_com womo womogo](https://github.com/google-ar/arcore-unity-extensions/assets/1468187/5dad8512-b209-49c0-9188-3d576d571a06)

**Comment by chiangandy** [comment]
Hi all, 

   Strange things happen sometimes, where the coordinates tilt or disappear when there is low light or indoors. How can I avoid this from happening?

![191629](https://github.com/google-ar/arcore-unity-extensions/assets/1468187/c9f124d1-7e29-4715-aeb5-c577b37677ac)

**Comment by JulesThuillier** [comment]
We are having the exact same problem, we toggle between an AR view and a Map view (the AR Session is disabled in Map View), and after a few toggle, the positions and orientations become very inaccurate. It seems that it is not the anchor position the problem but the device position.
I am looking for a fix...

**Comment by chiangandy** [comment]
During the time I tried to develop an app with arcore Geospatial, my
experience is Geospatial is not a stable solution even though their service
function is so amazing. Also, I found the community for ARCore has so few
people active on this. a lot of problems are unsolvable. So sad...

PS. I suggest you check Niantic and mapbox, they also have the same
solution.

REGARDS

江謝迪


Jules Thuillier ***@***.***> 於 2024年4月3日 週三 下午7:39寫道：

**Comment by JulesThuillier** [comment]
Thank you for your feebdack @chiangandy !
We are having the same experience here. It seems far from being production ready.
I have been looking into Niantic Lightship but it seems that it requires to scan what they call Public Locations before you can place an Mesh in AR. We need to be able to place it on pretty much any street (at least where Google Street is available).
But for other project we will test Niantic, it seems much stable and their "Private Locations" for development purposes are a great idea (it is a pain to always have to go out-side for testing, huge loss of time).

Have a beautiful day !

**Comment by chiangandy** [comment]
Exactly, we are follow the same situation. I strongly hope ARCore extension
can be a stable product even thought it request service charge(if it's not
expensive). But so far, it look like no this kind of future. :(
For Niantic, I am totally agree with your comments which  it has a little
bit different from Geo-spatial because of VPS with a scan mash. I am
curious why scanning mash is required even I have already know GPS
location.  But Niantic is more stable as I try.

by the way, did you check mapbox solution? maybe you will be interested on
that.

Mapbox AR demo
https://youtu.be/kBapWADpmgk?si=raMZ3lP_QZ3AV5Bb

江謝迪


Jules Thuillier ***@***.***> 於 2024年4月9日 週二 下午6:31寫道：

**Comment by chiangandy** [comment]
More information for you.
https://next.reality.news/news/mapbox-launches-location-based-ar-platform-with-multi-user-support-0183137/

Andy Chiang


Andy Chiang ***@***.***> 於 2024年4月12日 週五 下午2:30寫道：
